### PR TITLE
feat: adding node compatibility

### PR DESCRIPTION
**Pull request by AlexBHarley**
My webpack knowledge is horrible but with this and my other pull request (https://github.com/luniehq/cosmos-keys/pull/8) I'm able to `npm link` both these repositories and then write scripts using them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,12 +21,13 @@
...
     "babel-loader": "^8.0.5",
+    "babel-polyfill": "^6.26.0",
     "jest": "^24.7.1",
     "simsala": "^0.0.12",
     "webpack": "^4.31.0",
     "webpack-cli": "^3.3.2"
   },
   "dependencies": {
-    "axios": "^0.19.0"
+    "isomorphic-fetch": "^2.2.1"
```
Comment (faboweb) [inline package.json]: axios is usable across environments, why do you want to switch to isomorphic-fetch?

File: src/getters.js [hunk 2]
```diff
@@ -1,13 +1,12 @@
-"use strict"
-
+const fetchImpl = typeof fetch !== 'undefined' ? fetch : require('isomorphic-fetch');
```
Comment (jbibla) [inline src/getters.js]: we try to stay away from short forms / abbreviations - what does `fetchImpl` mean?

File: package.json (hunk 1 above)
Comment (faboweb) [reply package.json]: On a second thought, this makes the repo cleaner. 👍

Comment (AlexBHarley) [reply package.json]: Didn't see it being used anywhere, the API you're using is the `fetch` API, not axios.

File: src/getters.js (hunk 2 above)
Comment (AlexBHarley) [reply src/getters.js]: fetch implementation. Don't do any browser work usually so not sure what the best practice is for having different implementations across different environments.

File: package.json (hunk 1 above)
Comment (faboweb) [reply package.json]: Just noticed that aswell! Thanks for catching it.

**Comment by AlexBHarley** [comment]
Sorry about the force pushes, messed everything up this morning trying to get this library to work on a colleagues computer. Should be fine now

File: webpack.config.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
 const config = {
   devtool: "cheap-source-map",
-  entry: ['./src/index.js'],
+  entry: ['babel-polyfill', './src/index.js'],
```
Comment (faboweb) [inline webpack.config.js]: why did you add the polyfill here?

Comment (AlexBHarley) [reply webpack.config.js]: Couldn't get the regenerator-runtime stuff to work without it... will try again a little today to see if I can remove it

Comment (faboweb) [reply webpack.config.js]: I tried and removed the polyfill, but used it in the code where I use this lib:

```
import 'babel-polyfill';
const { createSignMessage } = require('@lunie/cosmos-api');
```

Comment (AlexBHarley) [reply webpack.config.js]: Ah yeah that makes sense. Will update this PR later today

File: webpack.config.js [hunk 4]
```diff
@@ -19,6 +20,16 @@ const config = {
         }
       }
     ]
-  }
+  },
+  externals: [
```
Comment (faboweb) [inline webpack.config.js]: Is it necessary to import `isomorphic-fetch` in node in parallel to this lib? If that's the case, could you maybe also add a description about this to the readme?

**Comment by AlexBHarley** [comment]
Alright let me know what you think... I can't get this to work without the `externals` property in the webpack config.

Not sure if the `import 'babel-polyfill'` is needed other places but I didn't need it in my tests locally.

**Comment by AlexBHarley** [comment]
@faboweb hoping to get this merged in soon, happy to do what it takes to make that happen. 

We're wanting to have a service signing transactions live soon and we'd love to not have to maintain our own fork.

**Comment by jbibla** [comment]
hey @AlexBHarley - @faboweb is out of office for a few days. thanks for your patience.

File: webpack.config.js (hunk 4 above)
Comment (faboweb) [reply webpack.config.js]: @AlexBHarley
